tests/test-maximize.lua

```diff
@@ -232,6 +232,30 @@ gears.table.merge(steps, {
 
         return true
     end,
+    -- Maximize then unmaximize. The window state should be restored.
+    function()
+        if #client.get() > 0 then return end
+
+        test_client(nil,nil,nil,nil,nil,{maximize_before=true,unmaximize_after=true})
+
+        return true
+    end,
+    function()
```

Comment:
This needs a `if #client.get() ~= 1 then return end` to avoid flackyness when the client takes more time to start for whatever reasons.

Reply:
I think `if not c then return end` will catch the flakiness, since #client.get() could only be 0 or 1 at this point.

Reply:
In other words, if `#client.get() ~= 1`, c would be `nil`, thus will this step will return and be retried.

Reply:
right, sorry for the noise.